glslc/test/option_dash_D.py

```diff
@@ -181,11 +181,11 @@ class TestDashCapDWithDashStdAndVersion(expect.ErrorMessage):
         """#version 310 es
         void main(){X}
         void foo(){Y}""", '.vert')
-    glslc_args = ['-DX=', '-DY=return 3;', '-std=450compatibility', shader]
+    glslc_args = ['-DX=', '-DY=return 3;', '-std=450core', shader]
```

Comment:
Changed `compatibility` to `core` since otherwise it errors out with:

```
#version: compilation for SPIR-V does not support the compatibility profile
```

This test's purpose is to test `-D` with `-std` and `#version`, it doesn't really matter whether we use `compatibility` or `core` here.